### PR TITLE
Examples: Add feedback example

### PR DESCRIPTION
**Pull request by timurvafin**
Adds functionality to send feedback emails to address specified via `FEEDBACK_EMAIL` environment variable.

* `Feedback` - model for validation, could be connected with DB
* `FeedbacksController`
* `ApplicationMailer#feedback`

### CODE REVIEW

File: app/models/feedback.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Feedback
+  include ActiveModel::Model
+
+  attr_accessor :email, :name, :message, :phone, :restaurant, :user
```
Comment (dmitrytrager) [inline app/models/feedback.rb]: looks like restaurant is redundant here

File: app/views/feedbacks/new.html.slim [hunk 2]
```diff
@@ -0,0 +1,25 @@
+- title("Contact Us")
+- description("Contact the team at Ofadaa")
```
Comment (dmitrytrager) [inline app/views/feedbacks/new.html.slim]: Flatstack maybe?)

**Comment by dmitrytrager** [comment]
:ok: except couple of comments
also, please provide description

**Comment by timurvafin** [comment]
Closed b/c of wrong pull destination -- should be `examples` branch instead of `master`

**Pull request closed without merging**